pkgs/by-name/st/steam-rom-manager/package.nix

```diff
@@ -10,11 +10,11 @@ appimageTools.wrapType2 rec {
   };
 
   extraInstallCommands = let
-    appimageContents = appimageTools.extract { inherit name src; };
+    appimageContents = appimageTools.extract { inherit pname version src; };
     in ''
-      install -m 444 -D ${appimageContents}/${name}.desktop -t $out/share/applications
-      substituteInPlace $out/share/applications/${name}.desktop \
-        --replace 'Exec=AppRun' 'Exec=${name}'
+      install -m 444 -D ${appimageContents}/steam-rom-manager.desktop -t $out/share/applications
```

Comment:
yes, this duplication is the right way to go

Reply:
I assume this is a compliment. if so, please mark it as resolved.